pkg/daemon/daemon.go

```diff
@@ -252,6 +253,47 @@ func NewClusterDrivenDaemon(
 	return dn, nil
 }
 
+const (
+	// IDs are taken from https://cgit.freedesktop.org/systemd/systemd/plain/src/systemd/sd-messages.h
+	sdMessageSessionStart = "8d45620c1a4348dbb17410da57c60c66"
+)
+
+func (dn *Daemon) detectEarlySSHAccessesFromBoot() error {
+	journal, err := sdjournal.NewJournal()
+	if err != nil {
+		return err
+	}
+	defer journal.Close()
+	if err := journal.AddMatch("MESSAGE_ID=" + sdMessageSessionStart); err != nil {
+		return err
+	}
+	if err := journal.SeekHead(); err != nil {
+		return err
+	}
+	r, err := journal.Next()
+	if err != nil {
+		return err
+	}
+	// if the journal cursor has something
+	if r == 1 {
```

Comment:
Minor: could avoid the indentation here by checking for 0 and returning early.

Reply:
thanks, fixed